public/_locales/zh_CN/messages.json

```diff
@@ -86,6 +86,18 @@
   "extensionOptionsExcludeFileTypesInformation": {
     "message": "留空则拦截任何类型的文件下载."
   },
+  "extensionOptionsTheme": {
+    "message": "Theme"
+  },
+  "extensionOptionsThemeLight": {
+    "message": "Light"
+  },
+  "extensionOptionsThemeDark": {
+    "message": "Dark"
+  },
+  "extensionOptionsThemeAuto": {
+    "message": "Auto"
+  },
```

Comment:
Hello @dream7180,

Could you translate this messages when you have time?

Thanks,
Baptiste C.